### ISSUE TITLE
json_config module not found

### ISSUE THREAD

**Issue by vandanrohatgi** [fossasia/susi_installer#103]
After successfully running the install.sh file I received a a module not found error.

To Reproduce
1. run ./install.sh
2. After whole process is complete and the terminal says 'enjoy'. run 
~/SUSI.AI/bin/susi-config set key=value ... 
 Desktop :
 - OS:ubuntu 18.04 LTS 
![issue](https://user-images.githubusercontent.com/43648786/69377525-798a6580-0cd2-11ea-98f7-1479edf622a1.png)

**Comment by norbusan** [comment]
Can you tell me your Python version? I assume that it is Python 3.8, which we need to fix support for ...

**Comment by norbusan** [comment]
Ahh, no ,... 18.04 LTS ... that is python 3.7

**Comment by vandanrohatgi** [comment]
> Can you tell me your Python version? I assume that it is Python 3.8, which we need to fix support for ...

It is python version 3.73

**Comment by norbusan** [comment]
Yeah, indeed. But I am not sure we can acutally run susi on 18.04 due to some dependencies (external). I need to check.

**Comment by vandanrohatgi** [comment]
Do tell If I can help!

**Comment by pranav1698** [comment]
I am using 18.04, but I am getting this error:
"Invalid input ('unknown key', 'key') ['/home/pranav/SUSI.AI/susi_installer/pythonmods/susi_config/__main__.py', 'set', 'key=value', '...']"
